docs/indexing.md

```diff
@@ -191,17 +192,43 @@ $ SET_URL=http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-production-
 ```
 You can see the progress of the SCSB indexing in sidekiq/Busy tab.  
 
-### Hook up your dev instance to the new index to see how it looks
+
+### Hook up catalog-qa to the new index to see how records are indexed
+
+Turn off sneakers in catalog-qa:
+- cd in your local princeton_ansible directory
+```
+$ pipenv shell   
+$ ansible orangelight_qa -u pulsys -m shell -a "sudo service orangelight-sneakers stop"
+```
+
+`$ ssh deploy@catalog-qa1`
+
+Use your preffered editor -nano or vi- and manually edit `app_configs/orangelight`  
```

Comment:
If you do this manually, isn't there a change that running the Orangelight playbook against QA would set it to the wrong value? Would it be better to change it in Princeton Ansible and run the Orangelight playbook against QA with the `site_config` tag?

Reply:
Using catalog-qa to review the index is not something critical. It's a suggestion so that we can do a QA and review the results at the and of the indexing process. You can use ansible to change it if you want but it should not be a permanent change; By default catalog-qa should be pointing to solr-staging and catalog-qa index.